src/index.js

```diff
@@ -0,0 +1,11 @@
+import React from "react"
+import ReactDOM from "react-dom"
+//component file
+import TodoContainer from "./components/TodoContainer"
+
+ReactDOM.render(
+  <React.StrictMode>
+    <TodoContainer />
+  </React.StrictMode>,
+  document.getElementById("root")
+)
```

Comment:
Hello, it would be a great idea to end this line with a semi-colon. Kindly have a look at list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) for a refresher.